Binance.Net/Objects/Models/Spot/Nft/BinanceNftTransaction.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Binance.Net.Objects.Models.Spot.NFT
+{
+    /// <summary>
+    /// NFT transaction
+    /// </summary>
+    public record BinanceNftTransaction
+    {
+        /// <summary>
+        /// Order number, 0: purchase order, 1: sell order, 2: royalty income, 3: primary market order, 4: mint fee
+        /// </summary>
+        [JsonPropertyName("orderNo")]
+        public string OrderNo { get; set; } = string.Empty;
+        /// <summary>
+        /// Tokens
+        /// </summary>
+        [JsonPropertyName("tokens")]
+        public IEnumerable<BinanceNftAsset> Tokens { get; set; } = Array.Empty<BinanceNftAsset>();
+        /// <summary>
+        /// Trade time
+        /// </summary>
+        [JsonPropertyName("tradeTime")]
+        public DateTime TradeTime { get; set; }
+        /// <summary>
+        /// Trade amount
+        /// </summary>
+        [JsonPropertyName("tradeAmount")]
+        public string TradeAmount { get; set; } = string.Empty;
```

Comment:
Assuming this is either a decimal or an int right?

Reply:
Yes, often API's return strings as decimal numbers with json can lead to some issues. But we should deserialize them to decimals so it's easier to work with.

Just changing the property type to decimal should be enough, the standard deserializer will handle the string parsing.